bundler/bundler.go

```diff
@@ -189,11 +191,22 @@ func (b *Bundler) BundleFromPEM(certsPEM, keyPEM []byte, flavor BundleFlavor) (*
 		}
 	}
 
-	certs, err := helpers.ParseCertificatesPEM(certsPEM)
+	certs, err := helpers.ParseCertificatesPEM(certsRaw)
 	if err != nil {
-		log.Debugf("failed to parse certificates: %v", err)
-		return nil, err
-	} else if len(certs) == 0 {
+		// If PEM doesn't work try DER
+		var keyDER crypto.Signer
+		certs, keyDER, err = helpers.ParseCertificatesDER(certsRaw, password)
```

Comment:
Do you have a test case where the incoming byte array is a set of more than one DER certificate?